### PR TITLE
bdfr: add page

### PR DESCRIPTION
**Pull request by Tonda39**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/bdfr.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# bdfr
+
+> Bulk downloader for Reddit.
+> More information: <https://github.com/Serene-Arc/bulk-downloader-for-reddit>.
+
+- Download videos/images from the specified [l]inks to URL or ID's of posts:
+
+`bdfr download {{path/to/output_directory}} -l {{post_url}} -l {{post_url}}`
```
Comment (spageektti) [inline pages/common/bdfr.md:8]:
```suggestion
`bdfr download {{path/to/output_directory}} -l {{post_url}}`
```

File: pages/common/bdfr.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`bdfr download {{path/to/output_directory}} -l {{post_url}}`
+
+- Download the maximum possible number (roughly 1000) of videos/images from a specified [u]ser:
+
+`bdfr download {{path/to/output_directory}} -u {{reddituser}} --submitted`
+
+- Download submission data (text, upvotes, comments etc.) [L]imited to 10 submissions for each [s]ubreddit (30 total):
+
+`bdfr archive {{path/to/output_directory}} -s 'Python, all, mindustry' -L 10`
```
[the remark this answers is not in the document]
Comment (Tonda39) [reply pages/common/bdfr.md:16]: Yeah, sorry my bad. I'm really bad at this xd.

File: pages/common/bdfr.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download videos/images from the specified [l]inks to URL or ID's of posts:
+
+`bdfr download {{path/to/output_directory}} -l {{post_url}}`
+
+- Download the maximum possible number (roughly 1000) of videos/images from a specified [u]ser:
+
+`bdfr download {{path/to/output_directory}} -u {{reddituser}} --submitted`
+
+- Download submission data (text, upvotes, comments etc.) [L]imited to 10 submissions for each [s]ubreddit (30 total):
```
Comment (spageektti) [inline pages/common/bdfr.md:14]:
```suggestion
- Download submission data (text, upvotes, comments, etc.) [L]imited to 10 submissions for each [s]ubreddit (30 total):
```

File: pages/common/bdfr.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+> Bulk downloader for Reddit.
+> More information: <https://github.com/Serene-Arc/bulk-downloader-for-reddit>.
+
+- Download videos/images from the specified [l]inks to URL or ID's of posts:
+
+`bdfr download {{path/to/output_directory}} -l {{post_url}}`
+
+- Download the maximum possible number (roughly 1000) of videos/images from a specified [u]ser:
+
+`bdfr download {{path/to/output_directory}} -u {{reddituser}} --submitted`
```
Comment (spageektti) [inline pages/common/bdfr.md:12]:
```suggestion
`bdfr download {{path/to/output_directory}} -u {{reddit_user}} --submitted`
```

**Pull request merged**